### ISSUE TITLE
[FEATURE] Mark eval images as dataset

### ISSUE THREAD

**Issue by subdavis** [Kitware/dive#936]
Post training, mark the eval vis images as a dataset so you can open and inspect them in the annotator:

https://viame.kitware.com/#/viewer/61453e8aa020b1e852638149

**Comment by waxlamp** [comment]
@subdavis, I'm a bit confused by this: your example link implies we can do this already? Or is the issue asking to cause the eval vis image to show up separately in the data browser (with its own `launch annotator` chip)?

**Comment by subdavis** [comment]
Yes, this is a feature request to add code to mark the output eval set as a dataset so the button appears.
